make/protogen.mk

```diff
@@ -217,6 +220,8 @@ endif
 		--proto_path=$(PROTO_BASE_PATH) \
 		--grpc-gateway_out=$(GATEWAY_M_ARGS_STR:%=%,)allow_colon_final_segments=true,logtostderr=true:$(GENERATED_BASE_PATH) \
 		$(dir $<)/*.proto
+	# Workaround for https://github.com/grpc-ecosystem/grpc-gateway/issues/229.
```

Comment:
Can you describe the issue you encountered?

Reply:
Not sure you wanted it as a comment here or in the protogen.mk file. I added it to the file.